cvat-core/src/annotations.js

```diff
@@ -211,6 +217,48 @@
         );
     }
 
+    // Annotations Metrics actions
+    function putAnnotationMetric(session, frame, user, labels) {
+        const cache = getCache('metrics');
+        const cacheKey = `${session.id}_${frame.id}_${user.id}`;
+        const frameAnnotation = cache.get(session)[cacheKey];
```

Comment:
Precisa checar se retorna algo no `cache.get(session)`, se voltar `undefined` vai ter exception no`[cachekey]`

```suggestion
        const frameAnnotation = cache.get(session) ? cache.get(session)[cacheKey] : null;
```

Reply:
É que o `getCache` garante que tem um **metrics** no cache e por isso não adicionei essa validação.

Reply:
Ah boa, verdade!